_sips/minutes/2020-01-27-sip-minutes.md

```diff
@@ -0,0 +1,127 @@
+month will be an opinionated executive summary of only the hightlights.)
+
+### Binary Integer Literal
+
+Thread: <https://contributors.scala-lang.org/t/pre-sip-support-binary-integer-literals/3559>
+
+The committee feels that, despite the benefits outlined in the proposal in this Pre-SIP, it doesn't fit the
+criteria for inclusion in Scala 3.0, specifically given it can be added later without technical breakages or
+changes in understanding of Scala ("book breaking").  However it might come up again when Dotty's `FromDigits`
+is discussed (in some future meeting).
+
+### Open Classes
+
+Thread: <https://contributors.scala-lang.org/t/sip-public-review-open-classes/3888>
+
+After Seb initially summarises the public review contributors' thread, some more technical details are
+discussed.
+
+One suggestion from the thread was that instead of being a new keyword it could be an `@open` annotation that
+users can opt-in to.  Martin responds that an `@open` annotation doesn't make the language any simpler, and
+it's a bit of a cop-out to use an annotation instead of a keyword, particularly as the principle is that
+annotations shouldn't change semantics (and its intended that `open` does).
+
+It's also highlighted that perhaps one of the biggest motivations for this proposal is to make the default
+`class Foo` be the right choice most of the time, countering the blog and conference talk advice that unless
+they're doing `final class Foo` (particularly `final case class Foo()`) then their code is "broken" or "not
+professional".
+
+Finally, a strawpoll was taken, with the committee members present, with the following results:
+  * Aye, by Martin
+  * Aye, by Sébastien
+  * Aye, by Guillaume
+  * Aye, by Adriaan
+  * Unsure, by Seth, but strongly in favour for just case classes
+  * Aye, by Josh, but with some nuance on some technical details
+
+### Explicit Nullability
+
+Thread: <https://contributors.scala-lang.org/t/sip-public-review-explicit-nulls/3889>
+
+Again, after some initial review of the thread feedback and chatting about some of that feedback, one of the
+highlights is that roughly Kotlin's platform types are very similar to Dotty's union type implementation, but
+that it would be worth comparing more closely to understand where they're different.
+
+The general update on the proposal is that there are some concerns, and both the proposal and the implementation
+require more work and more study.  Particularly there are implementation concerns with how unchecked null is
+handled, which has consequences on what code is consider valid.  As there are other paths to handle Java interop
+(including reviewing how platform types work), the proposal is going be somewhat withdrawn, so that the Dotty
+team can work on it some more and come back to this SIP proposal.
+
+Anyone interested in participating in experiementation is very welcome, as experimentation at this point would
```

Comment:
typo: experiementation

a run through a spellchecker would catch this and a few other of the typos